docs/source/reference/series.rst

```diff
@@ -93,3 +93,26 @@ Serialization / IO / Conversion
    Series.to_dict
    Series.to_clipboard
    Series.to_latex
+
+Datetime Methods
```

Comment:
take a look at how pandas does it for docs: https://raw.githubusercontent.com/pandas-dev/pandas/master/doc/source/reference/series.rst

They show up pretty nicely.

Reply:
Yeah let me play with it to see how far I can get. Will update here.

Reply:
I spent a couple of hours on this (basically trying to copy what pandas is doing) but looks like it involves quite bit of sphinx magic and it's going to take non trivial amount of time to figure out..I am not familiar with sphinx custom Documenter and templating which is what pandas is using for this.